pkg/prometheus/statefulset.go

```diff
@@ -319,6 +333,20 @@ func makeStatefulSetSpec(p v1alpha1.Prometheus, c *Config, ruleConfigMaps []*v1.
 			"-storage.tsdb.path=/var/prometheus/data",
 			"-storage.tsdb.retention="+p.Spec.Retention,
 		)
+
+		log := p.Spec.Log
```

Comment:
Why duplicate these arguments in this case? You should be able to just add all common flags below.